### PR TITLE
Redshift workspaces

### PR DESCRIPTION
**Pull request by pivnicek**
Passing tests, but minimal coverage.

### CODE REVIEW

File: tests/Backend/Workspaces/WorkspacesUnloadTest.php [hunk 1]
```diff
@@ -38,45 +39,59 @@ public function testCreateTableFromWorkspace()
...
-        $snowflakeConnection->query("insert into \"test.Languages3\" (\"Id\", \"Name\") values (1, 'cz'), (2, 'en');");
-
+        $db->query("insert into \"test.languages3\" (\"Id\", \"Name\") values (1, 'cz'), (2, 'en');");
 
         // create table from workspace
         $tableId = $this->_client->createTableAsyncDirect($this->getTestBucketId(self::STAGE_IN), array(
-            'name' => 'languages',
+            'name' => 'languages3',
             'dataWorkspaceId' => $workspace['id'],
-            'dataTableName' => 'test.Languages3',
```
Comment (Halama) [inline tests/Backend/Workspaces/WorkspacesUnloadTest.php]: @pivnicek why did yo changed the case? This is very important for testing varios edge cases.

File: tests/Backend/Workspaces/WorkspacesUnloadTest.php [hunk 2]
```diff
@@ -86,26 +101,38 @@ public function testCopyImport()
...
+            $db = new \PDO(
+                "pgsql:dbname={$connection['database']};port=5439;host=" . $connection['host'],
+                $connection['user'],
+                $connection['password']
+            );
+        } else throw new Exception("Unsupported backend for workspaces");
+
+
 
-        $snowflakeConnection->query("create table \"test.Languages3\" (
```
Comment (Halama) [inline tests/Backend/Workspaces/WorkspacesUnloadTest.php]: @pivnicek same here. this  have to be with all these specialities in table name.
